buildSrc/src/main/groovy/Libraries.groovy

```diff
@@ -0,0 +1,27 @@
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Dependency
+
+public class Libraries {
+    public final String junit = 'junit:junit:4.12'
```

Comment:
I'm just comparing this with the mentioned map we have. 
I only think that the map looks much cleaner and easier to manage. Here it is much verbose because it is Java. 

I was thinking if we can achieve what you want only with this: 

```
final junit = 'junit:junit:4.12'
```

without the types and public.